conans/client/installer.py

```diff
@@ -77,11 +77,11 @@ def _get_build_folder(self, conanfile, package_layout, pref, keep_build, recorde
         return build_folder, skip_build
 
     def _prepare_sources(self, conanfile, pref, package_layout, conanfile_path, source_folder,
-                         build_folder, package_folder):
+                         build_folder, package_folder, remotes):
```

Comment:
Does it make sense to store the `remotes` object as a member of the class `_PackageBuilder`? ...it is almost passed as an argument to every function.

Reply:
I am trying to keep the collaborators constructors with only other architectural collaborators, and pass all the data as arguments, towards a more functional approach and less OO.

Reply:
Oh! It is a `Remotes` object, not the `RemoteRegistry`... ok